kv/tikv/storage/kvstore/lock.go

```diff
@@ -0,0 +1,42 @@
+package kvstore
```

Comment:
I'm confused at the name

Reply:
Hmm, I don't think transaction is good because that might confuse mvcc transactions with TinySQL transactions. I agree that kvstore is not clear, but I think the problem is the organisation, not the name.

Is it OK to leave it for now, and I will split up the module and refactor later (once I've started on the latches)?

Reply:
okay